daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md

```diff
@@ -45,6 +45,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `queueName` | Y | Input/Output | The Service Bus queue name. Queue names are case-insensitive and will always be forced to lowercase. | `"queuename"` |
 | `ttlInSeconds` | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `"60"` |
 | `maxRetriableErrorsPerSec` | N | Input | Maximum number of retriable errors that are processed per second. If a message fails to be processed with a retriable error, the component adds a delay before it starts processing another message, to avoid immediately re-processing messages that have failed. Default: `10` | `10` |
+| `minConnectionRecoveryInSec` | N | Input | Minimum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. Default: `2` | `5` |
```

Comment:
For minConnectionRecoveryInSec and maxConnectionRecoveryInSec I like to also get these as values into the example YAML files (there are two in this article) and mark these as optional, so then people can see that they are choices. That way they see this copied Yaml files. Can you add these, if they are applicable in both yaml file example cases?

Reply:
Done